tests/sgl/device/test_device.cpp

```
@@ -9,6 +10,41 @@ using namespace sgl;
+    bool called{false};
+    NativeHandle callback_handle;
+};
+
+void SLANG_MCALL execute_callback_test(
+    const CommandNativeCallbackContext* context,
+    void* user_object,
+    const void* user_data,
+    size_t user_data_size
+)
+{
+    SGL_UNUSED(user_object);
```

Cppcheck (2.20.0)

[performance] Variable 'm_logger' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)